### DISCUSSION TITLE
ResBaz Venue Poll

### DISCUSSION THREAD

**Discussion by brandon-jernigan** [resbazaz/festival2024#19] [Polls]
Hello, everyone! As we move forward with planning ResBaz 2024, we're looking to finalize our venue choice. After thorough scouting and considering our event's requirements for a mix of formal and informal spaces, the Main Library Information Commons (lower level), stands out as a promising option. This setup offers us wide-open areas, plenty of natural light, external courtyard access, and a dedicated presentation room – ticking all boxes for a great conference/unconference venue. ResBaz was held here in 2018, and it seemed to work well, see pictures below

Map:
![image](https://github.com/resbazaz/festival2024/assets/12691918/f3ff1eb7-b375-4746-aa39-ccae3addf301)

ResBaz 2018:
![image](https://github.com/resbazaz/festival2024/assets/12691918/e63d5109-8622-451f-984a-8a17f1ddea10)
![image](https://github.com/resbazaz/festival2024/assets/12691918/947096cf-edfd-4357-91f6-5ead24ae660b)
![image](https://github.com/resbazaz/festival2024/assets/12691918/239aad6e-6260-4546-987a-4e7ad92bd8b3)

**Comment by val-pf** [comment]
Does this setup include only Tier 1, or Tier 2 (the 104 rooms) as well? I was under the impression only Tier 1 can be reserved.
I think this would be nice as an indoor/outdoor option, but think it will be harder to find than the Learning Studios at Catalyst on the main level.

**Reply by brandon-jernigan** [reply]
At least in 2018, Tier 2 was reserved for some events:
![image](https://github.com/resbazaz/festival2024/assets/12691918/7e42dd0b-cc86-49bb-a7aa-d721eef77a02)

Agreed on visibility, we should see if we can put a big sign by the stairs visible to people walking to and from the main library

**Reply by hidyverse** [reply]
Or even a cyverse tent on the mall with signs and directing traffic would be cool

**Comment by hidyverse** [comment]
Hi @brandon-jernigan can you add some info to this thread or post re: how you chose this space over the other library space?

**Comment by val-pf** [comment]
@jcoliver  can we have food and coffee down there?

**Reply by jcoliver** [reply]
Yah!
